docs/src/components/landingBlocks/HeroBlock.tsx

```diff
@@ -11,7 +11,7 @@ const { HeroPetsImage, HeroGraphicsImage } = Images;
 
 const content = {
   title: 'Ethereum development environment for professionals',
-  tagline: 'Flexible. Extensible. Fast.',
+  tagline: 'Flexible. Composable. Fast.',
```

Comment:
it worth to confirm this with the clients

Reply:
Ok. I'll do it at the next call with the customer. I don't think it's urgent